src/Device/deviceLister.ts

```diff
@@ -159,7 +116,7 @@ export const startWatchingDevices =
  * @returns {undefined}
  */
 export const stopWatchingDevices = () => {
-    // Start here
+    // Not sure, if this guard clause is really needed
```

Comment:
Seeing how the following command is wrapped in a try catch, it seems a bit unecceary? Also, I think checking if there is a hotplugTaskId seems like a better guard, and it would never exist without context.